### PR TITLE
fix: Add JavaneseIMDBClassification

### PR DESCRIPTION
**Pull request by ljvmiranda921**
Javanese IMDB classification dataset from https://huggingface.co/datasets/w11wo/imdb-javanese

## Checklist for adding MMTEB dataset

Reason for dataset addition: Add new language and dataset for classification (jav)

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [ ] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

File: mteb/tasks/Classification/jav/JavaneseIMDBClassification.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        type="Classification",
+        category="s2s",
+        date=None,
+        eval_splits=["test"],
+        eval_langs=["jav-Latn"],
+        main_score="accuracy",
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license=None,
```
Comment (shreeya-dhakal) [inline mteb/tasks/Classification/jav/JavaneseIMDBClassification.py:25]: It is "mit" license.

File: mteb/tasks/Classification/jav/JavaneseIMDBClassification.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        name="JavaneseIMDBClassification",
+        description="Large Movie Review Dataset translated to Javanese. This is a dataset for binary sentiment classification containing substantially more data than previous benchmark datasets.",
+        reference="https://github.com/w11wo/nlp-datasets#javanese-imdb",
+        dataset={
+            "path": "w11wo/imdb-javanese",
+            "revision": "11bef3dfce0ce107eb5e276373dcd28759ce85ee",
+        },
+        type="Classification",
+        category="s2s",
+        date=None,
```
Comment (shreeya-dhakal) [inline mteb/tasks/Classification/jav/JavaneseIMDBClassification.py:18]: Give a best estimate for the date.

File: mteb/tasks/Classification/jav/JavaneseIMDBClassification.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license=None,
+        socioeconomic_status="mixed",
+        annotations_creators="human-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation=None,
+        n_samples={"test": 25_000},
+        avg_character_length=None,
```
Comment (shreeya-dhakal) [inline mteb/tasks/Classification/jav/JavaneseIMDBClassification.py:32]: add avg_character_length

**Comment by ljvmiranda921** [comment]
Thanks for the reviews @shreeya-dhakal ! Updated :)

File: mteb/tasks/Classification/jav/JavaneseIMDBClassification.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        main_score="accuracy",
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="MIT",
+        socioeconomic_status="mixed",
+        annotations_creators="human-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation=None,
```
Comment (isaac-chung) [inline mteb/tasks/Classification/jav/JavaneseIMDBClassification.py:30]: Let's use the citation provided. 
```
@inproceedings{wongso2021causal,
    title={Causal and Masked Language Modeling of Javanese Language using Transformer-based Architectures},
    author={Wongso, Wilson and Setiawan, David Samuel and Suhartono, Derwin},
    booktitle={2021 International Conference on Advanced Computer Science and Information Systems (ICACSIS)},
    pages={1--7},
    year={2021},
    organization={IEEE}
}
```

Comment (isaac-chung) [reply mteb/tasks/Classification/jav/JavaneseIMDBClassification.py:30]: Actually, there are 2 citations at the bottom. Can you check which one is more related to the introduction of the dataset? That might change the dates as well.

**Review by isaac-chung — CHANGES_REQUESTED** [review]
Thanks for adding this. Just a small comment.

File: mteb/tasks/Classification/jav/JavaneseIMDBClassification.py (hunk 4 above)
Comment (ljvmiranda921) [reply mteb/tasks/Classification/jav/JavaneseIMDBClassification.py:30]: Hi thanks for catching! I think the one you provided is more related to the introduction of this dataset. Will update the PR in a few 👍

**Review by isaac-chung — APPROVED** [review]
Looks great! Thanks for iterating.

**Pull request merged**